state.md

```diff
@@ -0,0 +1,76 @@
+# State Pattern
+
+## Definition
+Erlaubt einem Objekt, sein Verhalten selbständig zu ändern, wenn sich sein interner Zustand ändern soll.
```

Comment:
please add a newline between headings and the textblocks. Take a look at the README.md, it links to a markdown linter

Reply:
done